hermes/apps/athena/modules/Earthfile

```diff
@@ -86,11 +87,23 @@ gen-bindings-http-proxy:
     # This allows the Rust code in http-proxy/src/ to import and use Hermes APIs
     SAVE ARTIFACT hermes.rs AS LOCAL athena/modules/http-proxy/src/hermes.rs
 
+builder-db:
+    DO rust-ci+SETUP
+
+    COPY wasm+wasi-src/wasi ./wasi
```

Comment:
This should _probably_ copy `hermes` into the `deps` directory, so that `athena` can use `hermes` as a dependency.